docs/runtime.md

```diff
@@ -0,0 +1,70 @@
+# Runtime
+
+Un runtime en pocas palabras y simplificado es un programa que interpreta código fuente y lo ejecuta. Para el desarrollo de nuestro proyecto se ha elegido el lenguaje de programación de TypeScript por lo que se puede optar por diferentes runtimes. Entre los runtimes para TypeScript se pueden encontrar los siguientes:
+
+- [Node.js](https://nodejs.org/en/)
+- [SWC](https://swc.rs/)
+- [Bun](https://bun.sh/)
+- [Deno](https://deno.com/)
+
+## Node.js
+
+Node.js es el runtime más popular para TypeScript. Es un runtime multiplataforma y de código abierto que se ejecuta en diferentes sistemas operativos. Node.js utiliza el motor de JavaScript V8 de Google para ejecutar código JavaScript. Además utiliza el ecosistema de paquete NPM.
+
+- Ventajas:
+    - Compatibilidad con gran cantidad de bibliotecas y frameworks debido a su popularidad.
+    - Comunidad muy activa, facilitando la busqueda de soluciones a problemas.
+
+- Desventajas:
+    - No es un runtime seguro debido a su historia de vulnerabilidades.
+    - No es un runtme rápido comparado con otros runtime como Deno o Bun.
+    - Es un runtime que necesita un gestor de depenedencias externo como NPM.
+    - Es necesario convertir el código TypeScript a JavaScript, también se puede usar ts-node para evitar esto sin embargo ts-node no es compatible con todas las características de TypeScript, además afecta al rendimiento ya que se compila en tiempo de ejecución.
+
+## SWC
+
+SWC es un compilador de TypeScript y JavaScript escrito en Rust. Sin embargo, también puede utilizarse como runtime para TypeScript. SWC es un proyecto de código abierto y multiplataforma. 
+
+- Ventajas:
+    - Es un runtime muy rápido comparado con otros runtimes como Node.js.
+    - Es un runtime tiene una alta seguridad debido a que está escrito en Rust.
+
+- Desventajas:
+    - No es tan popular como Node.js por lo que no tiene una comunidad tan grande.
+    - Es un runtime que necesita un gestor de dependencias externo.
+    - Igual que ocurría con Node.js, SWC necesita convertir el código TypeScript a JavaScript.
+    
+## Bun
+
+Bun es un runtime para TypeScript escrito en Rust. Es un proyecto de código abierto y multiplataforma.
+
+- Ventajas:
+    - Es un runtime diseñado para ser rápido siendo el más rápido de los runtime descritos.
```

Comment:
No pongáis una lista de ventajas y desventajas rándom. Lo único que interesa son cómo cumpla o no los criterios que establezcáis de antemano.

Reply:
Solucionado en 6002a8b92e3bf926c4c5fdb20f4baceb436a1113 se han mencionado solo la información relacionada con los criterios de selección